src/Components/UserProfile.js

```diff
@@ -1,24 +1,118 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components";
+import colors from "../constants/colors";
 import Button from "./Button";
 
+import users from "../data/users.json";
+
 const UserProfile = () => {
+  const [currentUser, setCurrentUser] = useState(null);
+  const [currentColor, setCurrentColor] = useState("craftBlue");
+
+  useEffect(() => {
+    setCurrentUser(users.data.getUsers[0]);
+  }, []);
+
+  if (!currentUser) {
+    return (
+      <UserProfileSection className="user-profile">
+        <SignInImage
+          className="not-signed-in-img"
+          // src={require(`../assets/`)}
+          alt="not signed in image"
+        />
+        <SectionHeader>
+          You're not signed in! Click the button below to sign in or create an
+          account.
+        </SectionHeader>
+        <Button name={"Login"} link={"/login"} />
+      </UserProfileSection>
+    );
+  }
+
   return (
-    <section className="user-profile">
-      <section className="bio">
-        <img className="profile-picture" src="" alt="crafter profile picture" />
-        <h2>Name</h2>
-        <h3>Contact info/ Location</h3>
-      </section>
-      <section className="available-items">
-        <h3>Supplies I am Offering</h3>
-        <Button name={"Add Craft"} link={"/contribution"} />
-        <div className="user-cards">craft cards would go here</div>
-      </section>
-    </section>
+    <UserProfileSection className="user-profile">
+      <BannerImg
+        className="not-signed-in-img"
+        src={require(`../assets/userProfileImgs/userBannerImg.jpg`)}
+        alt=""
+      />
+      <ProfilePicture
+        className="not-signed-in-img"
+        src={require(`../assets/userProfileImgs/CCProfilePic-01.png`)}
+        alt="your profile picture"
+        style={{ boxShadow: `-12px -12px 0px ${colors[currentColor]}` }}
+      />
+      <ProfileName style={{ color: `${colors[currentColor]}` }}>
+        {currentUser.name}
+      </ProfileName>
+      <ProfileEmail style={{ color: `${colors[currentColor]}` }}>
+        {currentUser.email}
+      </ProfileEmail>
+      <SectionHeader style={{ color: `${colors[currentColor]}` }}>
+        Crafts I'm Offering
+      </SectionHeader>
+      <CraftDisplaySection></CraftDisplaySection>
+      <Button name={"Add Craft"} link={"/contribution"} />
+      <SectionHeader style={{ color: `${colors[currentColor]}` }}>
+        Borrowed Supplies
+      </SectionHeader>
+      <CraftDisplaySection></CraftDisplaySection>
+      <Button name={"Log Out"} link={"/login"} />
+    </UserProfileSection>
   );
 };
```

Comment:
I'm really liking the structure of the profile page- very engaging and organized!